webclient/app/src/app/commons/alert/ErrorDialog.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import { useTranslation } from 'react-i18next';
+
+export default function ErrorDialog(props) {
```

Comment:
Why not merge ErrorDialog and ConfirmationDialog? Then you can input the severity via a prop